plugin.image.dumpert/addon.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+#
+# Imports
+#
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
+import os
+import sys
+import urllib.parse
+import xbmc
+import xbmcaddon
+
+LIB_DIR = xbmc.translatePath(
+    os.path.join(xbmcaddon.Addon(id="plugin.image.dumpert").getAddonInfo('path'), 'resources', 'lib'))
```

Comment:
no need to specify the id

Reply:
Removing the id gives me python 2 error:
20:33:50.986 T:2308   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: descriptor 'getAddonInfo' requires a 'xbmcaddon.Addon' object but received a 'str'
                                            Traceback (most recent call last):
                                              File "C:\Kodi_18.x_Portable_Test_Python2\portable_data\addons\plugin.image.dumpert\addon.py", line 17, in <module>
                                                os.path.join(xbmcaddon.Addon.getAddonInfo('path'), 'resources', 'lib'))
                                            TypeError: descriptor 'getAddonInfo' requires a 'xbmcaddon.Addon' object but received a 'str'
                                            -->End of Python script error report<--

Reply:
xbmcaddon.Addon().getAddonInfo("path")

Reply:
Ah, of course (doh!). Fixed.